ledger/streaming/client/Cargo.toml

```diff
@@ -5,22 +5,28 @@ authors = ["MobileCoin"]
 license = "MIT OR Apache-2.0"
 edition = "2018"
 
-[lib]
-
 [features]
 test_utils = []
 
 [dependencies]
+
 mc-common = { path = "../../../common" }
+mc-api = { path = "../../../api" }
+mc-consensus-scp = { path = "../../../consensus/scp" }
 mc-ledger-streaming-api = { path = "../api" }
 mc-transaction-core = { path = "../../../transaction/core" }
 mc-util-grpc = { path = "../../../util/grpc" }
 mc-util-uri = { path = "../../../util/uri" }
+mc-ledger-db = { path = "../../../ledger/db" }
 
 displaydoc = { version = "0.2", default-features = false }
 futures = { version = "0.3.21" }
 grpcio = { version = "0.10.0" }
-protobuf = "2.22.1"
+protobuf = { version = "2.22.1" }
+structopt = { version = "0.3" }
+serde_json = "1.0"
+tokio = { version = "1.17.0" }
+pin-project = {version = "1.0.10" }
```

Comment:
Nit: Sort alphabetically

Reply:
Will do, is there linter automation that can do this?

Reply:
Not that I know of

Reply:
Looks like https://crates.io/crates/cargo-sort fits the bill